### ISSUE TITLE
Loss function of LAN stage

### ISSUE THREAD

**Issue by PyoJunCode** [nagejacob/SpatiallyAdaptiveSSID#4]
In paper Eq(4), loss function of $\mathcal{L}_{LAN}$ is $(1-\alpha)||sg(\tilde{x}_1)-\tilde{x}_2||_1$.
But in code, it just using l1 for loss.
```
            # alpha = generate_alpha(BNN)
            # self.loss = self.criteron(BNN.detach() * (1 - alpha), LAN * (1 - alpha))
            self.loss = self.criteron(BNN, LAN)
```

What loss should I use in LAN stage?

**Comment by nagejacob** [comment]
Either will be ok.

**Comment by PyoJunCode** [comment]
Thanks for fast reply, closing this issue
